src/views/layouts/base.njk

```diff
@@ -42,7 +42,7 @@
 
 {% block bodyEnd %}
   {{ super() }}  
-    <script src="{{ assetPath }}/javascripts/all.js" type="text/javascript"></script>
+    <script src="{{ assetPath }}/js/all.js" type="text/javascript"></script>
```

Comment:
this works, but not sure how 🤔 no /js/ folder exists. Is there some magic or am I being silly?

Reply:
This line sets up a "virtual path prefix"
`app.use("/js", express.static("node_modules/govuk-frontend/govuk"));`

So the stuff in `node_modules/govuk-frontend/govuk` becomes available at `/js`

The docs explain it, about half way down the page:
https://expressjs.com/en/starter/static-files.html

Reply:
so do we need to remove our javascripts/all.js file? Thats whats confusing me

Reply:
Ah, right. I'm not sure what `javascripts/all.js` is. 
If it's identical to `govuk-frontend/govuk/all.js` then yeah, we probably don't need to be duplicating it.

Reply:
The gov.uk docs say:

>If you decide to add the JavaScript to your HTML, first either:
>
>- set up your routing so that requests for the JavaScript file are served from node_modules/govuk-frontend/govuk/all.js
> - copy the node_modules/govuk-frontend/govuk/all.js file into your  application

https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#javascript

But we seem to have been doing **both** of those things. So we just need to pick one.

Reply:
I think someone had an issue adding the node_modules version, so the local copy was created. I'd go for node_modules as you have done, so we recieve updates

Reply:
We were getting the JavaScript/all.js 
.map error. If you remember, if I did this incorrectly, sorry. 

But seemed we needed to map the node module govuk/all.is to the static public JavaScript/all.